### PR TITLE
Add matching error signatures to the 'impact function' field

### PR DESCRIPTION
**Pull request by ronniel1**

### CODE REVIEW

File: tools/add_triage_signature.py [hunk 1]
```diff
@@ -168,6 +169,18 @@ def update_ticket(self, url, issue_key, should_update=False):
...
 
+    def _add_signature_label(self, key, labels):
+        issue_labels = self._jclient.issue(key).fields.__dict__[CF_FUNCTION_IMPACT]
+        if issue_labels is None:
+            issue_labels = []
+        is_changed = False
+        for l in labels:
+            if l not in issue_labels:
+                is_changed = True
+                issue_labels.append(l)
```
Comment (omertuc) [inline tools/add_triage_signature.py:180]: The `for` loop can be replaced with:
```python
all_labels = set(issue_labels).union(labels)
if all_labels != issue_labels:
    self._update_fields(key, {CF_FUNCTION_IMPACT: all_labels})
```

**Comment by omertuc** [comment]
lgtm other than nit

File: tools/add_triage_signature.py (hunk 1 above)
Comment (ronniel1) [reply tools/add_triage_signature.py:180]: yea, I actually thought about doing something like this, but desided it is less efficient and less clear
If you insist I will change

Comment (omertuc) [reply tools/add_triage_signature.py:180]: I personally find for loops and flags much less clear, and clarity is more important than efficiency in code like this

Not insisting though, doesn't really matter

**Pull request merged**